app/src/main/java/org/mozilla/focus/activity/MainActivity.kt

```diff
@@ -43,10 +42,6 @@ open class MainActivity : LocaleAwareAppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
-        // Because some MVP characteristics are set from xml files we need to choose the mode of the theme
-        // according to MVP flag and not by system theme mode
-        AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_YES)
```

Comment:
This is temporary right? 🙂

Reply:
Yes,
![temporary-8ld3i6](https://user-images.githubusercontent.com/48995920/132546611-b0afe0ac-0b9b-4a86-9ecc-51647418e919.jpeg)